units/commit_queue.go

```diff
@@ -253,15 +255,12 @@ func getModules(ctx context.Context, githubToken string, nextItem *commitqueue.C
 			return nil, nil, true, errors.Wrapf(err, "module '%s' misconfigured (malformed URL)", mod.Module)
 		}
 
-		githash := ""
-		if mod.Issue != "" {
```

Comment:
This is never the case (it's required by the regex used to parse the modules) and doesn't make sense anyway (what's a patch without a place to get it from?)